packages/editor/styles/styles.css

```diff
@@ -520,6 +519,14 @@ p > *::selection {
   transition: transform 250ms ease;
 }
 
+*:not([dir="rtl"]).outline-list li > p::before {
+  left: -22px;
+}
```

Comment:
No need for this. You can reset the `left` property in the `rtl` CSS rule below.